concepticondata/conceptlists/Wu-2020-150.tsv

```diff
@@ -0,0 +1,151 @@
+Wu-2020-150-37	37	man	1554	MAN
+Wu-2020-150-38	38	egg	744	EGG
+Wu-2020-150-39	39	new	1231	NEW
+Wu-2020-150-40	40	three	492	THREE
+Wu-2020-150-41	41	white	1335	WHITE
+Wu-2020-150-42	42	I	1209	I
+Wu-2020-150-43	43	liver	1224	LIVER
+Wu-2020-150-44	44	hand	1277	HAND
+Wu-2020-150-45	45	rain	2108	RAINING OR RAIN
+Wu-2020-150-46	46	hide	2486	HIDE
+Wu-2020-150-47	47	tail	1220	TAIL
+Wu-2020-150-48	48	we	1212	WE
+Wu-2020-150-49	49	drink	1401	DRINK
+Wu-2020-150-50	50	louse	1392	LOUSE
+Wu-2020-150-51	51	snake	730	SNAKE
+Wu-2020-150-52	52	good	1035	GOOD
+Wu-2020-150-53	53	say	1458	SAY
+Wu-2020-150-54	54	small	1246	SMALL
+Wu-2020-150-55	55	fat	323	FAT (ORGANIC SUBSTANCE)
+Wu-2020-150-56	56	sun	1343	SUN
+Wu-2020-150-57	57	tree	906	TREE
+Wu-2020-150-58	58	cloud	1489	CLOUD
+Wu-2020-150-59	59	meat	634	MEAT
+Wu-2020-150-60	60	rock	668	ROCK
+Wu-2020-150-61	61	neck	1333	NECK
+Wu-2020-150-62	62	sand	671	SAND
+Wu-2020-150-63	63	wind	960	WIND
+Wu-2020-150-64	64	cold	1287	COLD
+Wu-2020-150-65	65	leaf	628	LEAF
+Wu-2020-150-66	66	dry	1398	DRY
+Wu-2020-150-67	67	earth	1228	EARTH (SOIL)
+Wu-2020-150-68	68	four	1500	FOUR
+Wu-2020-150-69	69	person	683	PERSON
+Wu-2020-150-70	70	go	695	GO
+Wu-2020-150-71	71	kill	1417	KILL
+Wu-2020-150-72	72	bite	1403	BITE
+Wu-2020-150-73	73	that	78	THAT
+Wu-2020-150-74	74	red	156	RED
+Wu-2020-150-75	75	burn	2102	BURN
+Wu-2020-150-76	76	mother	1216	MOTHER
+Wu-2020-150-77	77	road	667	ROAD
+Wu-2020-150-78	78	big	1202	BIG
+Wu-2020-150-79	79	sit	1416	SIT
+Wu-2020-150-80	80	father	1217	FATHER
+Wu-2020-150-81	81	long	1203	LONG
+Wu-2020-150-82	82	five	493	FIVE
+Wu-2020-150-83	83	mountain	639	MOUNTAIN
+Wu-2020-150-84	84	male	1553	MALE
+Wu-2020-150-85	85	what	1236	WHAT
+Wu-2020-150-86	86	knee	1371	KNEE
+Wu-2020-150-87	87	leg	1297	LEG
+Wu-2020-150-88	88	root	670	ROOT
+Wu-2020-150-89	89	soil	1228	EARTH (SOIL)
+Wu-2020-150-90	90	large	1202	BIG
+Wu-2020-150-91	91	grind	1033	GRIND
+Wu-2020-150-92	92	ashes	646	ASH
+Wu-2020-150-93	93	fall	1280	FALL
+Wu-2020-150-94	94	who	1235	WHO
+Wu-2020-150-95	95	right	1019	RIGHT
+Wu-2020-150-96	96	foot	1301	FOOT
+Wu-2020-150-97	97	house	1252	HOUSE
+Wu-2020-150-98	98	all	98	ALL
+Wu-2020-150-99	99	heavy	1210	HEAVY
+Wu-2020-150-100	100	back	1291	BACK
+Wu-2020-150-101	101	stand	1442	STAND
+Wu-2020-150-102	102	bad	1292	BAD
+Wu-2020-150-103	103	little	1246	SMALL
+Wu-2020-150-104	104	child	2099	CHILD
+Wu-2020-150-105	105	hot	1286	HOT
+Wu-2020-150-106	106	know	3626	KNOW
+Wu-2020-150-107	107	ten	1515	TEN
+Wu-2020-150-108	108	give	1447	GIVE
+Wu-2020-150-109	109	short	1645	SHORT
+Wu-2020-150-110	110	walk	1443	WALK
+Wu-2020-150-111	111	dead	1625	DEAD
+Wu-2020-150-112	112	female	1551	FEMALE
+Wu-2020-150-113	113	heart	1223	HEART
+Wu-2020-150-114	114	salt	1274	SALT
+Wu-2020-150-115	115	old	1229	OLD
+Wu-2020-150-116	116	hill	614	HILL
+Wu-2020-150-117	117	belly	1251	BELLY
+Wu-2020-150-118	118	sky	1732	SKY
+Wu-2020-150-119	119	laugh	1355	LAUGH
+Wu-2020-150-120	120	cut	1432	CUT
+Wu-2020-150-121	121	ash	646	ASH
+Wu-2020-150-122	122	close		
```

Comment:
Unmapped because it could be either SHUT or NEAR?

Reply:
Exactly